ansible/roles/sssd/vars/Rocky.yml

```diff
@@ -0,0 +1,14 @@
+---
+sssd_packages:
+  - 'sssd'
```

Comment:
On RL9, package providing sssd appears to be called `sssd-common`. There is no sssd package. And sssd-common, sssd-client, sssd-kcm, sssd-nfs-idmap appear to be installed by default in genericcloud image

Reply:
Hmm that isn't right, both packages are available on both OS. Installing sssd installs sssd-common. However at least on RL9 installing sssd-common appears to be enough for ldap auth.

RL9: installing sssd = 49M download
RL9: installing sssd-common = 3.9M download

```
[root@9bb7a8d6a6a5 /]# cat /etc/redhat-release 
Rocky Linux release 8.9 (Green Obsidian)
[root@9bb7a8d6a6a5 /]# dnf info sssd
Last metadata expiration check: 0:00:35 ago on Thu Sep 12 12:58:10 2024.
Available Packages
Name         : sssd
Version      : 2.9.4
Release      : 4.el8_10
...
Description  : Provides a set of daemons to manage access to remote directories and
             : authentication mechanisms. It provides an NSS and PAM interface toward
             : the system and a plug-gable back-end system to connect to multiple different
             : account sources. It is also the basis to provide client auditing and policy
             : services for projects like FreeIPA.
             : 
             : The sssd sub-package is a meta-package that contains the daemon as well as all
             : the existing back ends.
[root@9bb7a8d6a6a5 /]# dnf info sssd-common
Last metadata expiration check: 0:00:39 ago on Thu Sep 12 12:58:10 2024.
Available Packages
Name         : sssd-common
Version      : 2.9.4
Release      : 4.el8_10
...
Description  : Common files for the SSSD. The common package includes all the files needed
             : to run a particular back end, however, the back ends are packaged in separate
             : sub-packages such as sssd-ldap.